src/FSharp.Formatting.ApiDocs/GenerateHtml.fs

```diff
@@ -109,7 +109,9 @@ type HtmlRender(model: ApiDocModel, ?menuTemplateFolder: string) =
                   thead [] [
                       tr [] [
                           td [ Class "fsdocs-member-list-header" ] [ !!tableHeader ]
-                          td [ Class "fsdocs-member-list-header" ] [ !! "Description" ]
+                          td [ Class "fsdocs-member-list-header" ] [
+                              div [ Class "fsdocs-description-heading" ] [ !! "Description"; fsdocsDetailsToggle [] ]
+                          ]
```

Comment:
I guess the styling could probably just be applied to `.fsdocs-member-list-header` (which is currently unused), without needing to wrap these in a new `<div>`. Would that be better?

Reply:
It seems to work fine: d038a74.

The updated styling shouldn't change the display of any of the other heading cells with this class, since they each only have one child element: https://github.com/search?q=repo%3Afsprojects%2FFSharp.Formatting%20%22fsdocs-member-list-header%22&type=code